contrib/utilities/makeofflinedoc.sh

```diff
@@ -24,19 +24,22 @@ then
   exit 1
 fi
 
-mkdir -p images
+echo "Patching html files ..."
+sed -i 's#"https\?://www.dealii.org/images/#"images/#g' step_*.html class*.html
 
-echo "Downloading images (press ctrl-c to cancel) ..."
-cd images
-{
-trap "echo \"(skipping)\"" SIGINT
-wget -q -nd -A svg,png,gif,webm -m -l 1 -np https://www.dealii.org/images/steps/developer/
-}
-rm robots.txt*
-cd ..
+echo "Downloading images (this will take a long time; press ctrl-c to cancel) ..."
 
-echo "Patching html files ..."
-sed -i 's#"http://www.dealii.org/images/steps/developer/\(step-.*\)"#"images/\1"#g' step_*.html
-sed -i 's#"https://www.dealii.org/images/steps/developer/\(step-.*\)"#"images/\1"#g' step_*.html
+for i in steps/developer \
+         shape-functions/{DGPMonomial,DGPNonparametric,DGP}/P{1,2,3,4} \
+         shape-functions/{hierarchical,lagrange}/Q{1,2,3,4}; do
```

Comment:
You are hard-coding the list of folders here. The problem is that those need to be updated with every release. Wouldn't it make more sense to increase ``-l 1`` to a deeper level?

Reply:
Good point. Updated.